var/spack/repos/builtin/packages/singularity/package.py

```diff
@@ -24,29 +24,110 @@ class Singularity(SingularityBase):
     git = "https://github.com/hpcng/singularity.git"
     url = "https://github.com/hpcng/singularity/releases/download/v3.8.5/singularity-3.8.5.tar.gz"
```

Comment:
hpcng doesn't exist anymore, change to sylabs

Reply:
I think this is debatable, since https://github.com/hpcng/singularity redirects to apptainer. I'd just like to keep updating the two forks separately, and get rid (with some deprecation period) of this package.